TSC-Charter.md

```diff
@@ -17,60 +17,48 @@ In the case of the Node.js project, it is delegated to the Node.js
-TSC by a standard TSC motion. A TSC member may be removed from the
-TSC by voluntary resignation, by a standard TSC motion, or in accordance to the
-participation rules described below.
-
-Changes to TSC membership should be posted in the agenda, and may be suggested
-as any other agenda item.
-
-No more than one-fourth of the TSC members may be affiliated with the
-same employer. If removal or resignation of a TSC member, or a change of
-employment by a TSC member, creates a situation where more than
-one-fourth of the TSC membership shares an employer, then the situation
-must be immediately remedied by the resignation or removal of one or more
-TSC members affiliated with the over-represented employer(s).
-
-The TSC may, at its discretion, invite any number of non-voting observers to
-participate in the public portion of TSC discussions and meetings.
+membership beyond these rules. The TSC may add additional voting members to the
+TSC by a standard TSC motion. A TSC member can be removed from the
+TSC by voluntary resignation or by a standard TSC motion. A standard TSC motion
+can be used to change a regular TSC member to a voting TSC member, or to change
+a voting TSC member to a regular TSC member.
+
+No more than one-fourth of the TSC voting members may be affiliated with the
+same employer. If change in TSC voting membership or a change of
+employment by a TSC voting member creates a situation where more than
+one-fourth of the TSC voting membership shares an employer, then the situation
+must be immediately remedied by the removal of voting member status from one or
+more TSC voting members affiliated with the over-represented employer(s).
 
 The TSC shall meet regularly using tools that enable participation by the
 community (e.g. weekly on a Google Hangout On Air, or through any other
 appropriate means selected by the TSC). The meeting shall be directed by
 the TSC Chairperson. Responsibility for directing individual meetings may be
-delegated by the TSC Chairperson to any other TSC member. Minutes or an
+delegated by the TSC Chairperson to any other TSC voting member. Minutes or an
 appropriate recording shall be taken and made available to the community
 through accessible public postings.
 
-TSC members are expected to regularly participate in TSC activities.
-
-A TSC member is automatically removed from the TSC if, during a 3-month period,
-all of the following are true:
+TSC voting members are expected to regularly participate in TSC activities.
 
-* They attend fewer than 25% of the regularly scheduled meetings.
-* They do not participate in any TSC votes.
+A TSC voting member is automatically converted to a TSC regular member if
+they do not participate in three consecutive TSC votes.
 
 ## Section 4. Responsibilities of the TSC
 
-Subject to such policies as may be set by the CPC, the TSC is
+Subject to such policies as may be set by the CPC, the TSC voting members are
```

Comment:
I agree. Isn’t the process for any non-voting TSC member to become a voting TSC member to simply vote, or ask to become a voting member? So de facto the entire TSC is responsible for everything, because anyone can choose to “step forward” into voting status at any time. So we can just say “the TSC” for most things.

Reply:
Regular members can't vote: "Regular members can attend meetings and participate in TSC discussions, but do not vote.". The process to become a voting member for a regular member is outlined in the document: "A standard TSC motion can be used to change a regular TSC member to a voting TSC member, or to change a voting TSC member to a regular TSC member." And I think a standard TSC motion means lazy consensus over the whole TSC, and a vote (among the voting members) if no consensus can be reached? But if regular members are mostly inactive, does that mean we will be forced to move to a vote, or can we make a motion without the majority of the TSC participating?

Reply:
Is there a reason to require a motion for a non-voting member to become active again? I would think that we would want as low a bar (if any) as possible, so that no one feels any need to fight to stay as a voting member when they’ve missed votes and are automatically recategorized.

Reply:
With all the things tied to being a voting member (e.g. being an org admin), I’d say we do want a not too low bar.

Reply:
how about this clause instead of TSC motion for re-entry: `A TSC regular member is automatically converted to a TSC voting member if they participate in three consecutive TSC meetings`?

 - reasonable re-entry criteria that balances with exit criteria
 - meets TCS member's band-width / responsibility tradeoff
 - everyone knows the rules and easy to follow
 - scripts are easy to develop and run

Reply:
Note that this proposal doesn't take into account folks who might want to be TSC members without being  voting members (but maybe no one wants that, I don't know).
Personally, I'm -1 on any automation that adds folks as org admin without at least a TSC motion. If we can come up with a proposal that dissociates "being a voting TSC member" and "being an org admin", I would love that but that seems easier said than done.

Reply:
IMO, it is too late to change the text on this PR. It has been approved by the CPC and previously approved by the TSC. Coming in now to suggest changes is too late. It is already our charter. It just hasn't landed because a few other PRs need to land too to keep everything consistent.

If people want to change the charter further, that would be a new PR.